lib/config.js

```diff
@@ -0,0 +1,53 @@
+var nconf = require('nconf');
+var path = require('path');
+var fs = require('fs');
+
+class Configuration{
+
+        constructor(){
+            this.config = {};
+            this.init();
+        }
+
+        init() {
+            this.load();
+            this.print();
+        }
+
+        load() {
+            nconf.argv()
+            .env();
+
+            //required key check
+            nconf.required(['project', 'client_id', 'client_secret']);
```

Comment:
needs to check for the API url and the auth URL, too now.    Plus I have the impression the required check does not work at all.  
I now have a file  `.sphere-project-credentials` that contains `true:true:true`

Reply:
we don't need that file anymore, all the parameters are passed as command line parameters.
I have completely removed the `credentials` script.

Reply:
add the `apiUrl` & `authUrl` to the list of required parameters.